### ISSUE TITLE
Can't deploy precompiled binary

### ISSUE THREAD

**Issue by jin146** [marian-m12l/studio#20]
Hello, thank you for that fantastic work !
I can't deploy the app, even with openjdk-11 (Debian 9)
Here is the error message : 
```
ERROR StatusLogger No Log4j 2 configuration file found. Using default configuration (logging only errors to the console), or user programmatically provided configurations. Set system property 'log4j2.debug' to show Log4j 2 internal initialization logging. See https://logging.apache.org/log4j/2.x/manual/configuration.html for instructions on how to configure Log4j 2
java.lang.NoClassDefFoundError: com/lunii/device/gateway/raw/AbstractUsb4JavaLuniiDriver$DevicePluggedListener
	at studio.webui.MainVerticle.start(MainVerticle.java:55)
	at io.vertx.core.AbstractVerticle.start(AbstractVerticle.java:106)
	at io.vertx.core.impl.DeploymentManager.lambda$doDeploy$8(DeploymentManager.java:552)
	at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:320)
	at io.vertx.core.impl.EventLoopContext.lambda$executeAsync$0(EventLoopContext.java:38)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:404)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:462)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:897)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.ClassNotFoundException: com.lunii.device.gateway.raw.AbstractUsb4JavaLuniiDriver$DevicePluggedListener
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	... 11 more
nov. 30, 2019 12:54:08 PM io.vertx.core.impl.launcher.commands.VertxIsolatedDeployer
GRAVE: Failed in deploying verticle
java.lang.NoClassDefFoundError: com/lunii/device/gateway/raw/AbstractUsb4JavaLuniiDriver$DevicePluggedListener
	at studio.webui.MainVerticle.start(MainVerticle.java:55)
	at io.vertx.core.AbstractVerticle.start(AbstractVerticle.java:106)
	at io.vertx.core.impl.DeploymentManager.lambda$doDeploy$8(DeploymentManager.java:552)
	at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:320)
	at io.vertx.core.impl.EventLoopContext.lambda$executeAsync$0(EventLoopContext.java:38)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:404)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:462)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:897)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.ClassNotFoundException: com.lunii.device.gateway.raw.AbstractUsb4JavaLuniiDriver$DevicePluggedListener
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	... 11 more

java.lang.NoClassDefFoundError: com/lunii/device/gateway/raw/AbstractUsb4JavaLuniiDriver$DevicePluggedListener
	at studio.webui.MainVerticle.start(MainVerticle.java:55)
	at io.vertx.core.AbstractVerticle.start(AbstractVerticle.java:106)
	at io.vertx.core.impl.DeploymentManager.lambda$doDeploy$8(DeploymentManager.java:552)
	at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:320)
	at io.vertx.core.impl.EventLoopContext.lambda$executeAsync$0(EventLoopContext.java:38)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:404)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:462)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:897)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.ClassNotFoundException: com.lunii.device.gateway.raw.AbstractUsb4JavaLuniiDriver$DevicePluggedListener
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	... 11 more

```

I also tried compiling wit Maven 3, without success.

**Comment by marian-m12l** [comment]
Hello,

This looks like the lunii driver class is missing. Did you copy all the required jars from the luniistore? You also need to make sure the luniistore is up to date before copying the jars, as the driver classes recently changed. I cannot check right now, but it's also possible that the jars / driver classes changed again, and the application needs some adjustments.

Thanks for the feedback.

**Comment by jin146** [comment]
All the three required jar files were copied as written in the readme. I tried with the whole content of .local/share/Luniitheque without success.
The app version is 2.0, freshly installed today from the official website.

Looks like they updated the classes recently...

Note : I also had an strange output generating the json file in db folder.   `"error" : "Could not parse auth token."`
It doesn't look related but I indicate it anyway.

**Comment by marian-m12l** [comment]
Hi,

In the end, it wasn't a problem with the libraries, but with the classpath. Apparently I did not test the shell script... It should be fixed now with [version 0.1.1](https://github.com/marian-m12l/studio/releases/tag/0.1.1)

Regarding your error message for the metadata db, this looks like your authentication token is invalid. Did you log into the Luniistore recently ? If so, maybe check the content of the token with a tool like jwt.io ?

Please feel free to report any other issue, feedback is always welcome!

**Comment by jin146** [comment]
It works ! Thank you !
Great interface. It looks I can't scroll horizontally in editing mode though, it may be a limitation to my imagination :-)
